.github/workflows/pr_build.yml

```diff
@@ -68,6 +68,27 @@ jobs:
         run: |
           cd examples/powertools-examples-core/gradle
           ./gradlew build
+      - name: Setup Terraform
```

Comment:
```suggestion
      - name: Setup AWS credentials
        uses: aws-actions/configure-aws-credentials@5fd3084fc36e372ff1fff382a39b10d03659f355 # v2.2.0
        with:
          role-to-assume: ${{ secrets.AWS_ROLE_ARN_TO_ASSUME }}
          aws-region: ${{ env.AWS_DEFAULT_REGION }}
      - name: Setup Terraform
```

Reply:
you also need to add this after the env (line 56)
```
    permissions:
      id-token: write # needed to interact with GitHub's OIDC Token endpoint.
      contents: read
```

Reply:
Thanks, let me try again